renovate.json5

```diff
@@ -44,17 +44,17 @@
       {
         groupName: '@elastic/elasticsearch',
         packageNames: ['@elastic/elasticsearch'],
-        reviewers: ['team:kibana-operations'],
+        reviewers: ['team:kibana-operations', 'team:kibana-core'],
```

Comment:
@elastic/kibana-operations What is the correct format? I'd expect it to be `@elastic/kibana-operations`  and `@elastic/kibana-core` respectively

Reply:
https://docs.renovatebot.com/configuration-options/#reviewers
> Must be valid usernames. If on GitHub and assigning a team to review, use the prefix team:, e.g. provide a value like team:someteam.

LGTM

Reply:
Yeah, that's why I'm asking. `kibana-operations` and `kibana-core` aren't valid team names, are they?